patches/net/minecraft/server/players/PlayerList.java.patch

```diff
@@ -48,11 +48,38 @@
          ServerLevel serverlevel = p_11287_.serverLevel();
          p_11287_.awardStat(Stats.LEAVE_GAME);
          this.save(p_11287_);
+@@ -453,7 +_,14 @@
+         } else {
+             optional = Optional.empty();
+         }
+-
++        @Nullable
++        net.neoforged.neoforge.event.entity.player.PlayerRespawnEvent event = null;
```

Comment:
Why is this null at all?  The event can be fired unconditionally on the presence of `serverlevel1`.  It doesn't make sense to only allow the respawn to be changed if the original target dimension exists.